File: C/unit_3/Fa.c
#include <stdio.h>
int main()
{
    int cp, sp, res;
    printf("Enter the Cost Price of the Item : \n");
    scanf("%d", &cp);
    printf("Enter the Selling Price of the Item : \n");
    scanf("%d", &sp);
    if (sp > cp)
    {
        printf("Hurray!!! 😄 I got Profit of : %d", sp - cp);
    }
    else
    {
        printf("Oh no !!!😟 I got Loss of : %d", cp - sp);
    }
    return 0;
}

File: Java/unit_3/Fa.java
package Java.unit_3;

import java.util.Scanner;

public class Fa {
    public static void main(String[] args) {
        int cp, sp, res;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Cost Price of the item : \n");
        cp = sc.nextInt();
        System.out.println("Enter the Selling Price of the item : \n");
        sp = sc.nextInt();
        res = sp - cp;
        if (sp > cp)
            System.out.println("Hurray I got Profit of : " + res);
        else
            System.out.println("Oh I got Loss of : " + res);
        sc.close();
    }
}
